src/index.html

```diff
@@ -9,26 +9,76 @@
+                <ul class="dropdown-menu" ng-if="recentlyIndexed.length" ng-click="$event.stopPropagation()">
+                  <li ng-repeat="file in recentlyIndexed track by $index" style="display: block; cursor: pointer">
+                    <div style="margin-left: 10px">
+                      <a ng-click="getIndex(file)" style="float: left">{{file}}</a>
+                      <a ng-click="deleteIndex(file)" style="color: #c42940; float: right;" ng-show="recentlyIndexed[recentlyIndexed.indexOf(file)]">x</a>
+                    </div>
+                  </li>
+
+                  <li value="Delete All" ng-show="moreThanOneStored">
+                    <button ng-click="deleteIndex('Delete All')" style="width: 100%; background-color: #c42940; margin-top: 20px; border-radius: 0px;">Delete All</button>
+                  </li>
+                </ul>
+            </div>
 
         <button data-toggle="collapse" data-target="#about" id="aboutButton">About</button>
-          <div id="about" class="collapse">
-            <pre>
+          <div id="about" class="collapse" style="margin-top: 15px">
+            <p>
               An inverted index consists of a list of all the unique words that appear in any
               document, and for each word, a list of the documents in which it appears.
+            </p>
 
+          </p>
               This app takes an input JSON file (the JSON file should consist an array of objects, each
               object representing a document), and outputs a table showing which words appear in which
               document. You can also search for individual words in indexed documents.
-            </pre>
+            </p>
+          </div>
+        </div>
+
+        <div class="dropdown" id="dropdownSmall">
+          <div ng-show="indexInLocalStorage" class="dropdown right">
+          <button id="dropdown" class="btn btn-primary" data-toggle="dropdown"> <i class="fa fa-file"></i> </button>
+              <ul class="dropdown-menu" ng-if="recentlyIndexed.length" ng-click="$event.stopPropagation()">
+                <li ng-repeat="file in recentlyIndexed track by $index" style="display: block; cursor: pointer;">
+                  <div style="margin-left: 10px">
+                    <a ng-click="getIndex(file)" style="float: left">{{file}}</a>
+                    <a ng-click="deleteIndex(file)" style="color: #c42940; float: right;" ng-show="recentlyIndexed[recentlyIndexed.indexOf(file)]">x</a>
+                  </div>
+                </li>
+
+                <li value="Delete All" ng-show="moreThanOneStored">
+                  <button ng-click="deleteIndex('Delete All')" style="width: 100%; background-color: #c42940; margin-top: 20px; border-radius: 0px;">Delete All</button>
+                </li>
+              </ul>
+            </div>
+
+      </div>
+
+        <button id="littleAbout" data-toggle="collapse" data-target="#smallAbout" class="btn btn-primary">?</button>
```

Comment:
`littleAbout` doesn't seem to be a good name choice. You can name is better and still descriptive.